### ISSUE TITLE
How to use the onMainSlideClick event?

### ISSUE THREAD

**Issue by L-K-Mist** [Wlada/vue-carousel-3d#131]
Couldn't find an example where onMainSlideClick event is used.  Would you mind pasting an example snippet here?

**Comment by flavou45** [comment]
Hello,

Here is a link with an example of use: [https://jsfiddle.net/fx3azq9u/](https://jsfiddle.net/fx3azq9u/)

**Comment by Berkmann18** [comment]
I wonder why `:on-main-slide-click` works when the other event handlers are using `@`.

**Comment by gutisalex** [comment]
is it possible to get the index or any item that I have clicked on?

**Comment by kashyappiyush1998** [comment]
@gutisalex  I don't think you can pass index but you can get the index of the main slide - 
`<carousel-3d ref="carousel" :on-main-slide-click="onSelect">`
 ```

         onSelect() {

                let vm=this;
                console.log(this.$refs.carousel.currentIndex)
                    }
```

**Comment by Wlada** [comment]
Answered — example in the issue comments; the handler receives `{ index }`. Docs: https://wlada.github.io/vue-carousel-3d/api/
